magento/adobe-stock-integration
# Move default premium filter to di.xml configuration

### Description (*)
By default premium filter is set to All in \Magento\AdobeStockClient\Model\SearchParametersProvider\Premium::apply

Instead of setting the default filter in Client, it now sets by adding the filter to Magento\AdobeStockImage\Model\GetImageList in AdobeStockImage/etc/di.xml

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#471: Move default premium filter to di.xml configuration
2. ...

### Manual testing scenarios (*)
1. Configure Adobe Stock integration
2. Open Adobe Stock search
3. Verify premium filter all applied
4. Chose in grid filters filter premium
5. Verify filter applied.
